integration-tests/contract-estimation-tests.spec.ts

```diff
@@ -10,7 +10,7 @@ CONFIGS().forEach(({ lib, setup, knownBaker, createAddress, protocol, rpc }) =>
   const jakartanet = (protocol === Protocols.PtJakart2) ? test : test.skip;
   const kathmandunet = (protocol === Protocols.PtKathman) ? test : test.skip;
 
-  describe(`Estimate scenario using: ${rpc}`, () => {
+  describe(`Test estimate scenarios through contract api using: ${rpc}`, () => {
```

Comment:
We are testing the `estimate` not `through contract api`

Reply:
changed